src/iOS/toga_iOS/dialogs.py

```diff
@@ -16,6 +16,9 @@
 
 
 class TogaDialog(UIViewController):
+
+    interface = objc_property(object, weak=True)
```

Comment:
Any reason to add just `interface`, and not `impl`?

Reply:
Oh, I was confused about this one. Does the dialog actually have an "implementation" layer?

The following lines set `toga_iOS.dialogs.Dialog` as the *interface* layer:

https://github.com/beeware/toga/blob/e1650326e77d09a27f7e916072bb3379b334977f/src/iOS/toga_iOS/dialogs.py#L68-L70